wapitiCore/main/wapiti.py

```diff
@@ -504,7 +504,7 @@ async def attack(self, stop_event: asyncio.Event):
                             logging.info(f"{WAPITI_VERSION}. httpx {httpx.__version__}. OS {sys.platform}")
 
                         try:
-                            with open(traceback_file, "rb", encoding='utf-8') as traceback_byte_fd:
+                            with open(traceback_file, "rb") as traceback_byte_fd:
```

Comment:
Why did you remove `encoding` here ?

Reply:
it is not required as we are reading as binary

Reply:
It is not required as we are opening the file in binary mode